docs/website/docs/plus/features/transformations/dbt-transformations.md

```diff
@@ -0,0 +1,235 @@
+---
+title: dbt generator
+description: Generate dbt models automatically
```

Comment:
As we discussed yesterday (I think) we have the standalone dbt generator and the one that works in the project context. We will consolidate the commands I think, but for now we need to point this out to not confuse the users. This page demonstrated the standalone use, but all the stuff about how to add relationships etc apply to the project based one too.

Reply:
Is what was mentioned not enough?
<img width="1029" alt="Screenshot 2025-02-18 at 11 49 33" src="https://github.com/user-attachments/assets/20642775-5a1b-4b59-8dbe-13e7f4ba5543" />